fastchat/model/model_adapter.py

```diff
@@ -1989,6 +1989,16 @@ def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("metamath")
 
 
+class BagelAdapter(BaseModelAdapter):
```

Comment:
We do not need this class at all.
We can implement the `get_default_conv_template` in BaseModelAdapter.
```python
  def get_default_conv_template(self, model_path: str) -> Conversation:
      return get_conv_template(model_path)
```